lib/protocol/frame.js

```diff
@@ -1,6 +1,12 @@
 /**
  * Change focus to another frame on the page. If the frame id is null,
  * the server should switch to the page's default content.
+ * <example>
+    :Using `element` to find an iframe and providing it to `frame`
```

Comment:
this line is suppose to be the file name. To stay consistent with the other commands let's change that to be an file name and make the sentence to a comment